### PR TITLE
feat(patterns/fonts): implementing the new LM webfont

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [ ] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: packages/styles/components/_c.breadcrumb.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   @include set-font-scale('04', 'm');
 
   color: $color-font-darker;
-  padding: magic-unit-rem(0.875, 'true') 0 $mu100 magic-unit-rem(1.125, 'true');
+  padding: $mu100 0 magic-unit-rem(0.875, 'true') magic-unit-rem(1.125, 'true');
```
Comment (nherwegh) [inline packages/styles/components/_c.breadcrumb.scss:8]: links: 3px higher

**Review by nherwegh — CHANGES_REQUESTED** [review]
**Modals**
all text: 5px lower
link case: 2px higher

File: packages/styles/components/_c.tooltip.scss [hunk 2]
```diff
@@ -23,7 +23,7 @@
     border-radius: get-border-radius('m');
     box-sizing: border-box;
     color: $color-grey-000;
-    padding: $mu050 $mu100 magic-unit-rem(0.375, true);
+    padding: magic-unit-rem(0.4375, true) $mu100;
```
Comment (nherwegh) [inline packages/styles/components/_c.tooltip.scss:26]: tooltip text should be `2px higher`

File: packages/styles/components/_c.tabs.scss [hunk 3]
```diff
@@ -67,7 +67,7 @@
     cursor: pointer;
     display: block;
     outline: none;
-    padding: $mu125 $mu150 $mu100;
+    padding: magic-unit-rem(1.1875, true) $mu150;
```
Comment (nherwegh) [inline packages/styles/components/_c.tabs.scss:70]: tabs text should be `3px higher`

**Review by nherwegh — CHANGES_REQUESTED** [review]

**Comment by nherwegh** [comment]
@tiloyi `tabs` and `tooltips` are OK AY! 🎉

**Comment by nherwegh** [comment]
@tiloyi everything seems fine except the open accordion text `line-height` that should be large -> 1.5rem if I'm not wrong

**Review by caqueste — APPROVED** [review]

**Review by nherwegh — APPROVED** [review]

**Pull request merged**